src/test/scala-2/chiselTests/FormalContractSpec.scala

```diff
@@ -0,0 +1,95 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package chiselTests
+
+import chisel3._
+import chisel3.ltl._
+import chisel3.testers.BasicTester
+import chisel3.experimental.SourceLine
+import _root_.circt.stage.ChiselStage
+
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+
+import Sequence._
+
+class FormalContractSpec extends AnyFlatSpec with Matchers with FileCheck {
+  it should "support contracts with no operands" in {
+    class Foo extends RawModule {
+      val a = IO(Input(Bool()))
+      FormalContract {
+        AssertProperty(a)
+      }
+    }
+    generateFirrtlAndFileCheck(new Foo)("""
+      // CHECK-LABEL: module Foo
+      // CHECK: input a : UInt<1>
+      // CHECK: contract :
+      // CHECK:   intrinsic(circt_verif_assert, a)
+      """)
+  }
```

Comment:
Please include a test that the explicit `()` version errors, assuming we _do_ want to error in that case.

Reply:
Hmmm, that would be scala compiler error. Can we check for those?

Reply:
You're right. Not sure how to do that. Forget it!